### PR TITLE
Add lint script and use husky to lint on precommit

### PR DESCRIPTION
**Pull request by Vanish404**
everything is okey

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,9 +3,10 @@
   "version": "0.0.1",
   "description": "<h1><b>Plain-destination</b></h1>\r <h2>Structure</h2>\r | filename  | Content a file |\r | ------------- | ------------- |\r | index.html  | Main file in page-proofs  |\r | css/style.css  | Main styles  |\r | css/reset.css  | Reset CSS of Eric Mayer |\r | css/fonts.css  | Connect different the fonts |\r | css/media.css  | Main styles  |\r | js/script.css  | Work for element of form |",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+ "scripts": {
+   "lint": "eslint ./*.js js/*.js",
```
Comment (irnc) [inline package.json]: I guess `eslint .` would be enough.

File: package.json [hunk 2]
```diff
@@ -3,9 +3,10 @@
   "version": "0.0.1",
   "description": "<h1><b>Plain-destination</b></h1>\r <h2>Structure</h2>\r | filename  | Content a file |\r | ------------- | ------------- |\r | index.html  | Main file in page-proofs  |\r | css/style.css  | Main styles  |\r | css/reset.css  | Reset CSS of Eric Mayer |\r | css/fonts.css  | Connect different the fonts |\r | css/media.css  | Main styles  |\r | js/script.css  | Work for element of form |",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+ "scripts": {
```
Comment (irnc) [inline package.json]: Please check your indentation.

File: package.json [hunk 3]
```diff
@@ -3,9 +3,10 @@
   "version": "0.0.1",
   "description": "<h1><b>Plain-destination</b></h1>\r <h2>Structure</h2>\r | filename  | Content a file |\r | ------------- | ------------- |\r | index.html  | Main file in page-proofs  |\r | css/style.css  | Main styles  |\r | css/reset.css  | Reset CSS of Eric Mayer |\r | css/fonts.css  | Connect different the fonts |\r | css/media.css  | Main styles  |\r | js/script.css  | Work for element of form |",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (irnc) [inline package.json]: Do not remove `test`, this is out of scope for this task.

File: js/script.js [hunk 4]
```diff
@@ -99,17 +95,22 @@ function checkValues() {
   }
 }
 
+from.onkeyup = checkValues;
+to.onkeyup = checkValues;
```
Comment (irnc) [inline js/script.js]: Move it to the bottom, don't split from `submitButton.onclick`

**Comment by Vanish404** [comment]
@AlexeyKorkoza стоит ли пройтись eslint'ом по библиотеке https://github.com/pub-t/plain-destination/blob/master/libs/placeholders/placeholders.js , либо игнорить? - 300 ерорров

**Comment by AlexeyKorkoza** [comment]
@Vanish404 не надо.
